packages/web-app-files/src/components/SideBar/Actions/FileActions.vue

```diff
@@ -23,7 +23,7 @@ export default defineComponent({
   },
   setup() {
     const resource = inject<Resource>('resource')
-    const space = inject<SpaceResource>('space')
+    const space = inject<Ref<SpaceResource>>('space')
```

Comment:
here the injected space was correctly used (with `unref`) but the type was wrong.

Reply:
Could you fix the type for `resource` one line above as well while you're at it (same in `FileDetails.vue`)? 🙂

Reply:
Funny enough, the `resource` is provided as `Resource`, not as `Ref`. I don't know why we do that... but at least the type here is correct.

Reply:
Oh boy, you're right, see https://github.com/owncloud/web/blob/admin-settings-pagination/packages/web-app-files/src/components/SideBar/SideBar.vue#L296

We should align that some time in the future...